templates/digital-asset-template/move/Move.toml

```diff
@@ -1,9 +1,19 @@
 [package]
-name = 'my_app'
-version = '1.0.0'
-[dependencies.AptosFramework]
-git = 'https://github.com/aptos-labs/aptos-core.git'
-rev = 'main'
-subdir = 'aptos-move/framework/aptos-framework'
+name = "digital_asset_launchpad"
+version = "1.0.0"
+authors = []
+
 [addresses]
-module_addr='_'
\ No newline at end of file
+launchpad_addr = "_"
+minter = "_"
```

Comment:
what is it for? if required, we need to modify the move scripts probably

Reply:
this is required, it's like declaring all the addresses needed for the contract, `launchpad_addr` is address of contract itself, `minter` is address of dependency contract.

Reply:
> minter is address of dependency contract

then should it be a default address? or is it network based?

Reply:
it's network based